source/GooglePlayServicesProject.cshtml

```diff
@@ -260,23 +266,32 @@
 
   <ItemGroup>
     <!-- PackageReference -->
-    @foreach (var dep in @Model.NuGetDependencies) {
-      if (!dep.IsProjectReference) {
-      <PackageReference Include="@(dep.NuGetPackageId)" Version="@(dep.NuGetVersion)" PrivateAssets="none" />
+    @foreach (var dep in @Model.NuGetDependencies) 
+    {
+      if (!dep.IsProjectReference) 
+      {
+        @if (@Model.NuGetPackageId == "Xamarin.Firebase.Perf" || @Model.NuGetPackageId == "Xamarin.Firebase.Datatransport")
```

Comment:
What are these changes for?  Both of these packages list `androidx.annotation.annotation` in their POM so they should be picked up automatically.

Reply:
Nuget version package downgrade detection warnings/errors.

Let me see if I can remove that after cleanup.

Reply:
it works locally without that snippet. Let's wait CI